src/java/org/apache/cassandra/utils/logging/LogbackLoggingSupport.java

```diff
@@ -105,10 +100,9 @@ public void setLoggingLevel(String classQualifier, String rawLevel) throws Excep
         // if both classQualifier and rawLevel are empty, reload from configuration
         if (StringUtils.isBlank(classQualifier) && StringUtils.isBlank(rawLevel))
         {
-            JMXConfiguratorMBean jmxConfiguratorMBean = JMX.newMBeanProxy(ManagementFactory.getPlatformMBeanServer(),
-                                                                          new ObjectName("ch.qos.logback.classic:Name=default,Type=ch.qos.logback.classic.jmx.JMXConfigurator"),
```

Comment:
@nizhikov is this not available anymore? that mbean is gone?

Reply:
Hello, @smiklosovic 

Thanks for the feedback.

Yes, `JMXConfigurator` removed from loback.
Please, take a look into https://github.com/qos-ch/logback/commit/fa3de69304